platform/pulp/tasking/tasks.py

```diff
@@ -96,7 +96,7 @@ def _queue_reserved_task(name, inner_task_id, resource_id, inner_args, inner_kwa
     try:
         celery.tasks[name].apply_async(*inner_args, **inner_kwargs)
```

Comment:
I think we should avoid calling apply_async() with *something, or **something_else because our args and kwargs to the task are being mixed with the options we are passing to apply_sync itself. Say one of our tasks wants to use a keyword that is also used by apply_async like `max_retries`. Couldn't this be an issue?

It would be avoided (I think) by always having apply_async have parameters (positional and keyword) that are meant for the task itself being delivered via a keyword named `args` and `kwargs` to the apply_async().

Does that make sense? I'm still maybe a little confused. (It's complicated). :)